.github/workflows/python-package.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
From my opinion, I would say both do pretty much the same things (can be incorporated if anyone is interested, though in future it might need to separated again if needs more steps in python packging). But `python-package.yml` is a sort of check of the python packaging as python packaging in (PyPI) also follows `requirements.txt` to pack the package.

Reply:
I must confess, I need to learn about that a bit more about the that `python-package.yml`.

Reply:
It essentially runs pytest again, no?

Reply:
>  But python-package.yml is a sort of check of the python packaging as python packaging in (PyPI) also follows requirements.txt to pack the package.

Right, it is a bit more than just testing, but probably not super necessary. But can be kept for now.

Reply:
Right. As I do not have much knowledge and experience, I can not give a definite answer. You may already have noticed, that the checks run on the ´ci_cd` branch.

Reply:
Where does this come from? From the cookiecutter template?